src/SSVOpenHexagon/Core/HGScripting.cpp

```diff
@@ -19,6 +19,14 @@ namespace hg
 
 void HexagonGame::initLua_Utils()
 {
+    addLuaFn("u_getRandomSeed", //
+        [this] { return rng.get_seed(); })
+        .doc(
+            "Obtain a random seed for use in Lua scripting. This function "
+            "should always be used to initialize `math.randomseed` before "
+            "generating random numbers, as it will guarantee determinism "
+            "across replays and enable score validation.");
+
```

Comment:
I'm confused. Will this make a new seed every time or will it get the generated seed for that attempt specifically?

Reply:
The former. Let me clarify the docs...